### ISSUE TITLE
Why retrain at about 95 epoch with data aug disabled

### ISSUE THREAD

**Issue by DeppMeng** [tornadomeet/ResNet#47]
Hi, I am trying to reproduce the resnet imagenet result on mxnet.

I noticed that you have successfully reproduced the resnet [results](https://github.com/facebook/fb.resnet.torch) from facebook,
but you mentioned that 

> when epoch is about 95, cancel the scale/color/aspect augmentation during training, this can be done by only comment out 6 lines of the code, like this:

 I didn't quite understand why we need this procedure to get a similar results because I as far as I know Torch/PyTorch don't need this procedure.
Is that means if we don't apply this procedure, the result will be suffered (lower than the paper claimed)?

Thanks

**Comment by jamesdeep** [comment]
I have the same question, any clus till now?

> Hi, I am trying to reproduce the resnet imagenet result on mxnet.
> 
> I noticed that you have successfully reproduced the resnet [results](https://github.com/facebook/fb.resnet.torch) from facebook,
> but you mentioned that
> 
> > when epoch is about 95, cancel the scale/color/aspect augmentation during training, this can be done by only comment out 6 lines of the code, like this:
> 
> I didn't quite understand why we need this procedure to get a similar results because I as far as I know Torch/PyTorch don't need this procedure.
> Is that means if we don't apply this procedure, the result will be suffered (lower than the paper claimed)?
> 
> Thanks

I have the same question, any clus till now?

**Comment by ky-du** [comment]
@DeppMeng I think data augmentation may change the overall distribution of the dataset. When trained with small learning rate, the network may learn such distorted distribution.
  
By the way, do you have the training log / script on Pytorch? Thanks :)

**Comment by DeppMeng** [comment]
Sorry I am not doing research in this field now:)
